.github/workflows/approve-and-merge-dispatch.yml

```diff
@@ -0,0 +1,36 @@
+name: Approve and Merge Command Dispatch
+on:
+  issue_comment:
+    types: [created]
+jobs:
+  approveAndMergeDispatch:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Airbyte
+        uses: actions/checkout@v3
+      - name: Check PAT rate limits
+        run: |
+          ./tools/bin/find_non_rate_limited_PAT \
+            ${{ secrets.AIRBYTEIO_PAT }} \
+            ${{ secrets.OCTAVIA_GITHUB_RUNNER_TOKEN }} \
+            ${{ secrets.SUPERTOPHER_PAT }}
+
+      - name: Auto Approve Slash Command Dispatch
+        uses: peter-evans/slash-command-dispatch@v3
+        id: scd
+        with:
+          token: ${{ env.PAT }}
+          permission: write
```

Comment:
@supertopher this should limit who can invoke the bot (need to have at least write permission on this repo)